<ui version="4.0" >
 <class>GeneralInformationForm</class>
 <widget class="QWidget" name="GeneralInformationForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>General Information</string>
  </property>
  <widget class="QListWidget" name="listClusterPartitionsWidget" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>30</y>
     <width>181</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listOscarClustersWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>611</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>oscar</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>370</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title" >
    <string>Cluster Management</string>
   </property>
   <widget class="QWidget" name="verticalLayout_6" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>100</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveClusterInfoButton" >
       <property name="text" >
        <string>Save Cluster Configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Deploy Cluster</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>30</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title" >
    <string>Partition Information</string>
   </property>
   <widget class="QWidget" name="verticalLayout_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Partition name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="partitionNameEditWidget" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Linux distribution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="partitionDistroComboBox" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Number of nodes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="partitionNumberNodesSpinBox" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="virtualMachinesCheckBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Use virtual machines based on:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="virtualMachinesComboBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <item>
          <property name="text" >
           <string>Xen</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Qemu</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Setup as diskless</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OSCAR Clusters:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cluster Partitions:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>640</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QToolButton" name="addClusterButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/list-add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="removeClusterButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/list-remove.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>640</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QToolButton" name="addPartitionButton" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/list-add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="removeClusterButton_2" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/list-remove.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
